tests/util.go

```diff
@@ -137,7 +132,11 @@ func exec(r Result) {
 	if r.nameSpace != "" {
 		cmd = append(cmd, "-n", r.nameSpace)
 	}
-
+	if len(r.params) > 0 {
+		for _, v := range r.params {
+			cmd = append(cmd, "-p", v)
```

Comment:
there is `paramFlag` defined in contains.

Reply:
Done